src/Evolution/Systems/GeneralizedHarmonic/DampedHarmonicGaugeQuantities.hpp

```diff
@@ -0,0 +1,358 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Defines Functions for calculating damped harmonic gauge quantities from
+/// 3+1 quantities
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/Tags.hpp"  // IWYU pragma: keep
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "Time/Tags.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;  // IWYU pragma: keep
+/// \endcond
+
+// IWYU pragma: no_forward_declare Tags::Time
+// IWYU pragma: no_forward_declare Tags::Coordinates
+// IWYU pragma: no_forward_declare Tags::deriv
+// IWYU pragma: no_forward_declare Tensor
+// IWYU pragma: no_forward_declare db::ComputeTag
+// IWYU pragma: no_forward_declare gr::InverseSpatialMetric
+// IWYU pragma: no_forward_declare gr::Tags::Lapse
+// IWYU pragma: no_forward_declare gr::Tags::Shift
+// IWYU pragma: no_forward_declare gr::Tags::SpacetimeMetric
+// IWYU pragma: no_forward_declare gr::Tags::SpacetimeNormalOneForm
+// IWYU pragma: no_forward_declare gr::Tags::SpacetimeNormalVector
+// IWYU pragma: no_forward_declare gr::Tags::InverseSpatialMetric
+// IWYU pragma: no_forward_declare gr::Tags::SqrtDetSpatialMetric
+
+namespace GeneralizedHarmonic {
+namespace DampedHarmonicGaugeHelpers {
```

Comment:
Reasoning for the above comment:
Since you wrote these functions only for computing DampedHarminicGauge without the intention of exposing them outside, these would go into a detail namespace.
If on the other hand you want to make them reusable for other gauge choices, you should move them to another more general file/namespace, which may have "Helpers" in its name.

This is how we understand the distinction between the namespace suffixes `detail` and `Helpers`.  @nilsdeppe, can you confirm this?